charts/pega/config/deploy/prconfig.xml

```diff
@@ -15,4 +15,6 @@
         for this node.  This is useful for settings which are specific to this node and should not be shared by multiple nodes on this
         system. -->
 
+        <env name="prconfig/cluster/hazelcast/v4/enabled" value="true" type="java.lang.String"/>
+
```

Comment:
@saikarthik528 Could you please add a short comment on the usage of this setting?

Reply:
added comment about usage